src/AdminConsentRequiredController.js

```diff
@@ -0,0 +1,104 @@
+/*!
+ * Copyright (c) 2020, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import hbs from 'handlebars-inline-precompile';
+import { _, View, loc } from 'okta';
+import FormController from 'util/FormController';
+import FormType from 'util/FormType';
+import ScopeList from 'views/admin-consent/ScopeList';
+
+const ConsentModel = {
```

Comment:
we don't Model's and View's in separate files in V1 ?

Reply:
there isn't a strict pattern as okta-ui modules. It embedded Model/Form in Controller file at many places.